app/styles/variables.css

```diff
@@ -110,12 +110,15 @@
   --color-green-8: #108944;
   --color-green-9: #06371b;
 
-  --color-red-0: #fcc5d8;
   --color-red-1: #a82a2a;
   --color-red-2: #c23030;
   --color-red-3: #f31260;
   --color-red-4: #f4256d;
-  --color-red-5: #fcc5d8;
+  --color-red-5: #f881ab;
```

Comment:
Goes without saying, this does not affect anything else. As I've said, I'm gonna go through this more thoroughly, but I really needed some more shades of red.

Reply:
These new "reds" are very pink. I suppose light red colors look a bit pink, but it might make more sense to just call them pinks